stockflux-securities-master/src/components/AddSecurityButton.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./AddSecurityButton.css";
+
+const AddSecuritiesButton = () => {
+  return (
+    <Link to="/inputform/">
+      <div className="add-security-button">
+        <button>Click to add security</button>
```

Comment:
Can we change this button text to just read "Add a security"?

Reply:
Yep changed now